libpod/define/container_inspect.go

```diff
@@ -571,13 +571,13 @@ type InspectAdditionalNetwork struct {
 type InspectNetworkSettings struct {
 	InspectBasicNetworkConfig
 
-	Bridge                 string               `json:"Bridge"`
-	SandboxID              string               `json:"SandboxID"`
-	HairpinMode            bool                 `json:"HairpinMode"`
-	LinkLocalIPv6Address   string               `json:"LinkLocalIPv6Address"`
-	LinkLocalIPv6PrefixLen int                  `json:"LinkLocalIPv6PrefixLen"`
-	Ports                  []ocicni.PortMapping `json:"Ports"`
-	SandboxKey             string               `json:"SandboxKey"`
+	Bridge                 string                          `json:"Bridge"`
+	SandboxID              string                          `json:"SandboxID"`
+	HairpinMode            bool                            `json:"HairpinMode"`
+	LinkLocalIPv6Address   string                          `json:"LinkLocalIPv6Address"`
+	LinkLocalIPv6PrefixLen int                             `json:"LinkLocalIPv6PrefixLen"`
+	Ports                  map[string][]ocicni.PortMapping `json:"Ports"`
```

Comment:
I don't think this is correct - should be a `struct{HostIp string, HostPort string}` instead of an `ocicni.PortMapping`